src/components/Welcome.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Welcome() {
+  return (
+    <div>
+      <h1>Welcome to your Smart Shopping List!</h1>
+
+      {/* For accessibility reasons this is a link, but can be styled as button */}
+
```

Comment:
Hi Indira, we were following the [wireframe ](https://github.com/the-collab-lab/tcl-13-smart-shopping-list/blob/main/_resources/wireframes/welcome-screen.png) that's in the resources and it has both a create list and join list feature, however we will refactor so that the create list feature is only rendered conditionally. Thank you for pointing this out!

Reply:
We've refactored so that create list and view list are rendered conditionally based on the token. Thank you for your detailed feedback!